### PR TITLE
Fix code examples in auto-completion layer

### PR DESCRIPTION
**Pull request by j4**

### CODE REVIEW

**Comment by robbyoconnor** [comment]
Hi, thanks for your PR -- you might want to close this and make another pull request against a topic branch as it may be some time before this is merge

File: layers/auto-completion/README.org [hunk 1]
```diff
@@ -118,8 +118,8 @@ The variable has no effect when =auto-complete= is used.
 
 #+BEGIN_SRC emacs-lisp
 (setq-default dotspacemacs-configuration-layers
-  '(auto-completion :variables
-                    auto-completion-enable-sort-by-usage t))
+  '((auto-completion :variables
+                    auto-completion-enable-sort-by-usage t)))
 #+END_SRC
```
Comment (robbyoconnor) [inline layers/auto-completion/README.org]: Why? The previous was correct.

File: layers/auto-completion/README.org [hunk 2]
```diff
@@ -105,8 +105,8 @@ To enable docstring tooltips set =auto-completion-enable-help-tooltip= to =t=
 
 #+BEGIN_SRC emacs-lisp
 (setq-default dotspacemacs-configuration-layers
-  '(auto-completion :variables
-                    auto-completion-enable-help-tooltip t))
+  '((auto-completion :variables
+                    auto-completion-enable-help-tooltip t)))
 #+END_SRC
```
Comment (robbyoconnor) [inline layers/auto-completion/README.org]: Why, the previous was correct?

File: layers/auto-completion/README.org [hunk 3]
```diff
@@ -97,7 +97,7 @@ the popup, set the variable =auto-completion-enable-snippets-in-popup= to =t=.
 #+BEGIN_SRC emacs-lisp
   (setq-default dotspacemacs-configuration-layers
                 '((auto-completion :variables
-                                   auto-completion-enable-snippets-in-popup t))
+                                   auto-completion-enable-snippets-in-popup t)))
```
Comment (robbyoconnor) [inline layers/auto-completion/README.org]: What should be done here is

```emacs-lisp
'(auto-completion :variables 
                              auto-completion-enable-snippets-in-popup t)
```

**Comment by robbyoconnor** [comment]
The only one that needs to be fixed is the outer-most  group of parens needs to go. Everything else doesn't fix this things at all.

File: layers/auto-completion/README.org (hunk 1 above)
Comment (StreakyCobra) [inline layers/auto-completion/README.org]: @robbyoconnor No, the new one is correct. You need parenthesis around the layer when there are `:variables`.

File: layers/auto-completion/README.org (hunk 2 above)
Comment (StreakyCobra) [inline layers/auto-completion/README.org]: @robbyoconnor No, the new one is correct. You need parenthesis around the layer when there are `:variables`.

File: layers/auto-completion/README.org (hunk 3 above)
Comment (StreakyCobra) [inline layers/auto-completion/README.org]: @robbyoconnor No, the new one is correct. You need parenthesis around the layer when there are `:variables`.

**Comment by StreakyCobra** [comment]
@robbyoconnor See my comments above.

@j4 :+1: If there is something that could be better, it is the commit message. Something like `Fix code examples in auto-completion layer` would have been more explicit :-) If you are used to git and know how to amend a commit and force a push, feel free to do it. If not, don't worry, we will do it when merging your code :relaxed:

**Comment by robbyoconnor** [comment]
@j4 you should definitely consider using a topic branch -- this might now be merged in immediately

File: layers/auto-completion/README.org (hunk 3 above)
Comment (robbyoconnor) [inline layers/auto-completion/README.org]: nevermind :)

**Comment by StreakyCobra** [comment]
@j4 Using the develop branch will prevent you to work on a new PR in the meantime this one get merged. As it's documentation it will probably be merged soon so don't worry. For more explanation about the typical github workflow, you can read @travisbhartwell blog: http://iam.travishartwell.net/2015/07/29/how-to-make-a-pr-for-spacemacs/ that explain step by step how to contribute to Spacemacs :smiley: (by the way #3898 is also speaking about this)

**Comment by TheBB** [comment]
Thanks! Cherry-picked in develop. You can safely delete your branch.

**Pull request closed without merging**